onnxruntime/contrib_ops/cpu/signal/dft.h

```diff
@@ -1,15 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#ifdef BUILD_MS_EXPERIMENTAL_OPS
```

Comment:
>ef BUILD_MS_EXPERIMENTAL_OPS [](http://example.com/codeflow?start=4&length=28)

Seems lost

Reply:
Using function bodies instead of registering kernels.